docs/HowTo/Logging and Errors.md

```diff
@@ -0,0 +1,168 @@
+# Logs & Errors Consolidation
+Although there is sufficient logging in Quorum (geth), the error messages are not always straightforward to decipher.
+This is our take on cataloguing of error messages along with possible cause and remediation actions, to serve as a first point of reference before reaching out to the support team.
+
+!!! note
+    The log level “ERROR” is written out in uppercase as part of the log message and can be used for alert monitoring.
+
+| Message & Parameters | Cause | Action |
+| -------------------- | ----- | ------ |
+| "Could not search for pattern", "pattern", pattern, "contract", contracts[types[i]], "err", err | Generated by `abigen` due to an internal error when generating ABI for a contract |  *Raise an issue ticket* |
```

Comment:
Maybe write the original error message between back ticks ` so that we clearly see the exact message match what the user had on the console and also to prevent some parts to be interpreted as markdown. Remove backticks around program names like abigen. If the goal it to emphasis it, use bold or italic. Best would be a link to the relevant doc part. Last Action part may not need any italic style, normal test may be ok IMO.
Exemple for this line:
```suggestion
| `"Could not search for pattern", "pattern", pattern, "contract", contracts[types[i]], "err", err` | Generated by [abigen](https://link_to_abigen_doc) due to an internal error when generating ABI for a contract |  Raise an issue ticket |
```